packages/core/src/env/consts.ts

```diff
@@ -5,3 +5,6 @@ export const isProduction = getEnv('NODE_ENV') === 'production';
 export const port = Number(getEnv('PORT', '3001'));
 export const mountedApps = Object.freeze(['api', 'oidc']);
 export const developmentUserId = getEnv('DEVELOPMENT_USER_ID');
+
+// Trusting TLS offloading proxies: https://github.com/panva/node-oidc-provider/blob/main/docs/README.md#trusting-tls-offloading-proxies
+export const trustingTlsOffloadingProxies = getEnv('TRUSTING_TLS_OFFLOADING_PROXIES') === 'true';
```

Comment:
What about just be `PROXY`

Reply:
After my consideration, the value of `proxy` is commonly treated as an url. just leave it as what it is now.